MigHerCas/be-my-creator-next
# Create custom video player

## Summary
Videos are important for Be My Creator. That's why we need a reliable generic component ready to use. 

## Details
* Native HTML5 video element is the chosen option
* VideoModal component wraps it 
* A11y is a priority
* Poster image as a loading fallback

## Testing
* Check how every device plays videos using this component
* Check how every device renders VideoModal and its content

## References

* Inner PR #3 

## Checklists

- [ ] Tested in local (Frontend: Chrome, Safari, Firefox, Mobile)
- [x] I reviewed my own Pull Request commit by commit
- [x] I didn't just select everything, this PR really does abide by these ^

## Screenshots

![image](https://user-images.githubusercontent.com/29380502/208616245-cab2d858-a929-43ee-af48-e01e58918c97.png)
